rabbitmq.nuspec

```diff
@@ -32,8 +27,4 @@ The RabbitMQ Chocolatey package source has a new home at https://github.com/tnwi
     </dependencies>

```

Comment:
I wasn't sure how to check to see if the version should be updated. I just installed OTP 19 manually for testing.